exercises/practice/largest-series-product/.meta/config.json

```diff
@@ -0,0 +1,17 @@
+{
+  "authors": [],
```

Comment:
You should put your Github username here as the exercise author to get authoring credit / extra rep on the Exercism website.